beta-diversity.rmd

```diff
@@ -0,0 +1,338 @@
+---
+title: "Beta-diversity"
+output: html_document
+---
+
```

Comment:
Could you add here:
```{r setup, echo=FALSE, results="asis"}
library(rebook)
chapterPreamble()
```

Reply:
How about the "_bookdown.yml"?

Reply:
i have edited on the same file it should be there

Reply:
The "Files changed" tab above does not show any changes in "_bookdown.yml"? Have you committed the changes?

Reply:
fixes have been made at branch issue_34 #34